README.md

```diff
@@ -1,3 +1,3 @@
 # The Linux Mint Welcome screen
 
-![Screenshot of Mint 19.3](/screencaptures/welcome_screen_mint_19_3.png?raw=true)
+![Screenshot of Mint 19.3](/screencaptures/welcome_screen_mint_21_1.png?raw=true)
```

Comment:
Update the alt-text to 21.1 also please.

Reply:
Sorry I didn't see it

It is now updated